packages/venia-ui/lib/components/Header/cartTrigger.css

```diff
@@ -8,3 +8,12 @@
     font-weight: 600;
     margin-left: 0.3rem;
 }
+
+.icon--empty {
+    composes: root from '../Icon/icon.css';
+}
+
+.icon--filled {
```

Comment:
So instead of passing a color prop, if you want to override the filled color you would provide your own `icon--filled` class, right?